src/util/graph.creator.ts

```diff
@@ -35,9 +35,7 @@ export class GraphCreator {
     }
 
     private static prepareFileContent(fileContent: string): string {
-        return fileContent
-            .replace(/[/][*](.|\n|\r)*?[*][/]/g, '') // Strip block-comments
-            .replace(/export(?!([ ]+(const|class|function|interface|type|default|enum|let|abstract)[ ]))(.|\n|\r\n)+?from[ ]/g, ''); // Strip export-from statements;
```

Comment:
The beautiful regex 😭
Just thought, it might be worth mentioning that we explicitly also want to catch export-from-statements; it's not that intuitive - maybe worth a comment at the place where we do the "from"-matching?
(no need to do that in this PR though)